### PR TITLE
Created the works and about me section

### PR DESCRIPTION
**Pull request by Leeoasis**
The following is an outline of the tasks I performed:

>Added the  works section with a display of the past projects completed.

> Added the About me section.

> Added the styling for both the works section and the about me section.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -61,6 +62,180 @@ <h2 class="icon">LET'S CONNECT</h2>
...
+              </ul>
+                <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+           </div>
+            <div  class="first_buttons">
+              <a href="#">html</a>
+              <a href="#">css</a>
+              <a href="#">Javascript</a>
+            </div>
+            <div class="last-button">
+              <button>See Project</button>
```
Comment (Zuhaib042) [inline index.html:89]: - [x] This button should have a `:hover` and pressed state `:active` that change its color corresponding to the Figma design. you can check this link out and learn how to use these pseudo-classes. 
https://www.w3schools.com/cssref/sel_active.php
- [ ] Kindly consider applying these pseudo-classes per design guidelines to other `see project` buttons.

Figma design guidelines
po

File: index.html [hunk 2]
```diff
@@ -61,6 +62,180 @@ <h2 class="icon">LET'S CONNECT</h2>
...
+                  <li class="canopy">CANOPY</li>
+                   <li>Back End Dev</li>
+                   <li>2015</li>
+              </ul>
+                <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+           </div>
+            <div  class="first_buttons">
+              <a href="#">html</a>
+              <a href="#">css</a>
+              <a href="#">Javascript</a>
```
Comment (Zuhaib042) [inline index.html]: - [ ] Kindly consider using a list `ul` tag here and each of these technologies should be a `li`.
- [ ] Kindly consider applying these changes across the cards as I can see the same pattern is repeating elsewhere.
- The list of project technologies (e.g., "JavasScript", "HTML" and "CSS") are a list ul and each of them a li

File: index.css [hunk 3]
```diff
@@ -76,3 +74,162 @@ body {
 .social-icons li {
   list-style-type: none;
 }
+
+/* ----------Works page styling begins here----- */
+
```
Comment (Zuhaib042) [inline index.css:80]: - [ ] Kindly consider styling the Section of works with `display: grid` as it is a project requirement.
- The main container of the projects has a `display:grid`

File: index.html [hunk 4]
```diff
@@ -61,6 +62,180 @@ <h2 class="icon">LET'S CONNECT</h2>
...
+            </div>
+            <div class="last-button">
+              <button>See Project</button>
+           </div>
+          </div>
+      </div>
+      <div class="flex-container">
```
Comment (Zuhaib042) [inline index.html]: - [ ] This must be a `section` tag as mentioned in the requirements. kindly consider applying this change.
- The container of the section is a `section` tag

File: index.html [hunk 5]
```diff
@@ -61,6 +62,180 @@ <h2 class="icon">LET'S CONNECT</h2>
...
+          <div class="works-card-2">
+              <h2 class="works-title">About Myself</h2>
+              <P class="about-me">Hello I'm a software developer I can help you build a project, feature or website. Look through some of my work and experience! If you like what you see and have a project you need coded don't hesitate to contact me.</P>
+              <p class="icon">LET'S CONNECT</p>
```
Comment (Zuhaib042) [inline index.html]: - [ ] This must be a `h` header tag as this is for best semantic practices.
- If there are other titles (e.g., "Let's connect") use a header tag `hx`

File: index.html [hunk 6]
```diff
@@ -61,6 +62,180 @@ <h2 class="icon">LET'S CONNECT</h2>
...
+                    ><img class="g-social-media" src="static/images/github.png" alt="github"
+                    /></a>
+                 </li>
+                 <li>
+                    <a href="#"
+                    ><img class="v-social-media" src="static/images/Vector (4).png" alt="vector"
+                    /></a>
+                  </li>
+                </ul>
+              <button class="resume">Get my resume</button>
```
Comment (Zuhaib042) [inline index.html:203]: - [ ] Kindly consider styling this button with hover and active pseudo-class with corresponding design guidelines of colors.
- The button (e.g., "Get my resume") is a link `a` or `button` tag, and follow the button styling: Active, Hover, and Pressed (see design)

File: index.css [hunk 7]
```diff
@@ -76,3 +74,162 @@ body {
...
+/* -------------- About page styling begins here -------- */
+
+.about_page {
+  height: 100vh;
+}
+
+.works-card-2 {
+  background-color: #fff;
+  display: flex;
+  flex-direction: column;
```
Comment (Zuhaib042) [inline index.css:172]: - [ ] This section is not similar to the Figma design. Kindly consider exporting the background image of about section in Figma design and apply here in about section as `background-image`.
- [ ] Also consider applying button styling changes here as I mentioned  in the `see project` button.
you design
asd

figma design
asdf

File: index.html [hunk 8]
```diff
@@ -61,6 +62,180 @@ <h2 class="icon">LET'S CONNECT</h2>
             /></a>
           </li>
         </ul>
-      </section>
+    </section>
+    <section class="grid-container">
+      <div class="flex-container">
+        <div class="works-card">
```
Comment (Zuhaib042) [inline index.html:68]: - [ ] Kindly consider applying the following changes to your design to make it pixel-perfect to Figma design:
- [ ] The body of your project should have `#E5E5E5` color as seen in figma design.
- [ ] The `li`s under the title `CANOPY` are not properly aligned. kindly align them per figma design and also in other projects' cards.
- [ ] The `p` text in the Figma design has a lighter font weight. kindly consider fixing it.
your design
qwer


figma design
qw

File: index.css [hunk 9]
```diff
@@ -76,3 +74,162 @@ body {
...
+  border-radius: 8px;
+  padding: 10px;
+  background-color: #f7f7f9;
+  gap: 12px;
+  margin: 12px;
+}
+
+.languages_wrapper {
+  padding: 12px 24px;
+}
```
Comment (Zuhaib042) [inline index.css]: - [ ] This section of languages and Skills is not consistent with Figma design and overlapping. kindly consider changing it to make it similar to the Figma design as it is a project requirement.
your design
zx


figma design
zxcv

**Review by Zuhaib042 — CHANGES_REQUESTED** [review]
Hi @Leeoasis , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
close

## Highlights
✅ linters passing
✅ Professional PR

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._
- [ ] kindly consider writing commit messages in Imperative sentences. for example instead of writing `Added staying to ...`, you can write `Add styling to ...`

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by roshan-bajgain — COMMENTED** [review]
Hi @Leeoasis, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that has a linter error please fix it and submit it again.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

**Review by RaoAkif — APPROVED** [review]
## STATUS: APPROVED 🎉🎉🎉

Hi @Leeoasis,

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

approve-image

## Highlights
✔ Linters are on point 
✔ Splendid work with the README file
✔ Great use of GitHub flow
✔ Good code design and implementation

## Optional suggestions
- This design has been either disturbed or was earlier not considered.
Suggestions:
 ```
 .languages_wrapper {
    padding: 12px 0px;
}

.first_buttons, .coding-languages ul {
    list-style: none;
    padding-left: 0px;
}
```
But keep in mind, that other things might not break.

image

- _Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**